Added Lightning Talks Documentation (#65)
kossiitkgp/docs: events/lightning-talks.md

> # Lightning Talks

Reviewer: Short form for Lightning Talks (LT) can be written in brackets here.

> ## What is LT?

Reviewer: Try to write LT in full here.

- An event where CTMs present about a topic chosen by them, be it tech-related or not.

> - Each presentation should be of around 15 min

Reviewer: Please add fullstops everywhere for consistency.

## Purpose of the Event

> - To motivate CTMs to enhance their speaking and presentation skills (After all, they would be the ones conducting upcoming workshops)

Reviewer: ```suggestion
- To motivate CTMs to enhance their speaking and presentation skills, as they would be the ones conducting upcoming workshops.
```

- To encourage healthy discussion among CTMs.

## Before the Event

> - The event can last for 4-6 hours, so time can be divided into 2 days accordingly.

Reviewer: Add a point - schedule slots between CTMs in advance.

- Room (with AV support) must be booked for the event.

## After the Event
Upload the following in the [kossitkgp/slides](https://github.com/kossiitkgp/slides) repo

> - Recording of the event, if its recorded

Reviewer: I believe this point should not be under `slides`. If recorded, the event should probably be uploaded to YT after discussion and approval on Slack.

Author: In `slides` repo, in the README file, we have inserted YT links. That's why mentioned that here. Should I remove those links as well?

Reviewer: It's fine if the YT links are in slides repo, but it may not always be recorded or made public, and the way this is worded, it feels like the recording should be directly added to the repo XD.

Author: oh sorry, i'll improve that

Reviewer 2: Is it fine now?

Author: i'm working on the recordings one, else is fine i guess
